en/news/_posts/2020-12-25-ruby-3-0-0-released.md

```diff
@@ -120,7 +120,7 @@ It uses [async](https://github.com/socketry/async) which provides the event loop
 
 ## Static Analysis
 
-> 2010s were an age of statically typed programming languages. Ruby seeks the future with static type checking, without type declaration, using abstract interpretation. RBS & TypeProf are the first step to the future. More steps to come. --- Matz
+> 2010s were an age of statically typed programming languages. Ruby seeks the future with static type checking, without type declaration, using abstract interpretation. RBS & TypeProf are the first steps to the future. More steps to come. --- Matz
```

Comment:
```suggestion
> 2010s were an age of statically typed programming languages. Ruby seeks the future with static type checking, without type declaration, using abstract interpretation. RBS & TypeProf are the first step to the future. More steps to come. --- Matz
```

I wouldn't alter the quotation since both versions are feasible (they both together form the first step; more to come).